### ISSUE TITLE
Feature Request: vomnibar to incognito

### ISSUE THREAD

**Issue by haxwithaxe** [philc/vimium#2495]
It would be super convenient to have a way to summon the vomnibar in a normal browser tab and have it open the search/url in an incognito.
Example workflow:
1. keycombo to get specially targeted vomnibar.
2. normal vomnibar things.
3. incognito window pops up with the result instead of the normal tab/window.

**Comment by philc** [comment]
Makes sense, although this is quite niche, so I don't think we'll support it. A workaround is to configure Vimium to work in incognito windows, open a new one, and then use the vomnibar there.
